lib/Net/Google/Drive/Simple/V3.pm

```diff
@@ -9,24 +9,33 @@ use parent qw< Net::Google::Drive::Simple::Core >;
 use URI ();
 use URI::QueryParam ();
 use File::Basename ();
+use JSON qw< from_json >;
 use Log::Log4perl qw(:easy);
 
 use constant {
     'HTTP_METHOD_GET'    => 'GET',
     'HTTP_METHOD_POST'   => 'POST',
+    'HTTP_METHOD_PUT'    => 'PUT',
     'HTTP_METHOD_PATCH'  => 'PATCH',
     'HTTP_METHOD_DELETE' => 'DELETE',
 
+    'HTTP_CODE_OK'     => 200,
+    'HTTP_CODE_RESUME' => 308,
+
     'TYPE_STRING'  => 'string',
     'TYPE_INTEGER' => 'integer',
     'TYPE_LONG'    => 'long',
     'TYPE_BOOLEAN' => 'boolean',
     'TYPE_OBJECT'  => 'object',
     'TYPE_BYTES'   => 'bytes',
 
-    'SIZE_5MB'     => 5_242_880, # 5M
+    'SIZE_256K'   => 262_144,          # default chunk size
+    'SIZE_5MB'    => 5_242_880,        # low limit
+    'SIZE_5120GB' => 5_497_558_138_880 # high limit
 };
 
+use constant { 'DEF_CHUNK_SIZE' => SIZE_5MB() * 2 }; # 10 MB
```

Comment:
maybe define a `SIZE_1MB` so you can reuse it in more locations :-)

Reply:
Where else would it be used?

Reply:
`SIZE_5MB()` and `DEF_CHUNK_SIZE()` would then be easier to read and be able to use it

Reply:
I think their names are sufficient.